src/pedalmarking.ts

```diff
@@ -19,11 +18,11 @@ function L(...args: any[]) {
  * at the coordinates `x` and `y. Takes into account the glyph data
  * coordinate shifts.
  */
-function drawPedalGlyph(name: string, context: RenderContext, x: number, y: number, point: number): void {
-  const glyphData = PedalMarking.GLYPHS[name];
-  const glyph = new Glyph(glyphData.code, point, { category: 'pedalMarking' });
-  // Center the middle of the glyph with the middle of the note head (Tables.STAVE_LINE_DISTANCE / 2)
-  glyph.render(context, x - (glyph.getMetrics().width - Tables.STAVE_LINE_DISTANCE) / 2, y);
+function drawPedalGlyph(name: string, ctx: RenderContext, x: number, y: number): void {
+  const glyph = new Element(PedalMarking.CATEGORY);
+  glyph.setText(PedalMarking.GLYPHS[name] || name);
```

Comment:
Effectively the same, but I think we should use `??` operator when we are checking for undefined / null. We should keep `||` for actual boolean expressions (e.g., in an if / else if condition).

Reply:
done